### ISSUE TITLE
Zabbix Triggers - 'host in maintenance' does not work

### ISSUE THREAD

**Issue by michelmarinho** [alexanderzobnin/grafana-zabbix#355]
#### I'm submitting a ...
- [X] Bug report
- [ ] Feature request
- [ ] Question / Support request

Grafana 4.1.1
Zabbix 3.2.0 and 2.4.5
Zabbix Plugin 3.3.0
CentOS 7.2 /Debian 8

Allways show all hosts.

**Comment by alexanderzobnin** [comment]
Ok, I'll test it.

**Comment by alexanderzobnin** [comment]
@michelmarinho are you sure? I checked on my Zabbix 3.0 and it works. I use `maintenance=false` option for `trigger.get` api method and there wasn't any changes in 3.2. 
https://www.zabbix.com/documentation/3.2/manual/api/reference/trigger/get

**Comment by michelmarinho** [comment]
@alexanderzobnin I'm sorry, but for some reason, Grafana was not using "maintenance = false" in any moment. I did the restart service and it worked. I also did the restart on the plugin upgrade. I went in history and confirmed it. Any way, the plugin is OK in versions 2.4.5 and 3.2. Thank you for this.

root@:~# history | grep grafana
   50  grafana-cli
   51  grafana-cli  plugins ls
   52  grafana-cli  plugins
   53  grafana-cli  plugins update-all
   54  systemctl restart grafana-server
  506  systemctl restart grafana-server
  510  history | grep grafana
root@~#
